smarts/sstudio/genhistories.py

```diff
@@ -430,7 +510,7 @@ class OldJSON(_TrajectoryDataset):
     history files to the new .shf format."""
 
     @property
-    def rows(self):
+    def rows(self) -> Generator[Tuple, None, None]:
```

Comment:
I think we will want to deprecate `OldJSON` unless we are supporting migrations. It is additional work to keep servicing it.

Reply:
Good idea.  I'll add a comment to that now.